app/src/androidTest/java/com/example/dailytaskmanager/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example.dailytaskmanager;

```

Comment:
There's nothing here. Remove this file.

Reply:
Similarly added both androidTest and test directory of /app/src in .gitignore along with .idea